hamilton/plugins/h_spark.py

```diff
@@ -1087,6 +1089,29 @@ def new_callable(**kwargs) -> DataFrame:
             input_types={upstream_name: DataFrame},
         )
 
+    @staticmethod
+    def create_drop_node(
```

Comment:
Other possibility is for this to depend on the input dataframe as well and ask it for its columns, then add them to select. This seems clearer though.